### DISCUSSION TITLE
Template temporaneo

### DISCUSSION THREAD

**Discussion by gbvitrano** [opendatasicilia/opendatasicilia.it#22] [Annunci]
Ho installato un template temporaneo in wordpress, ancora non ottimizzato,  mi passo il tempo a impostare una nuova struttura. 
Adesso che ho modificato il file host del pc  come faccio a visualizzare il vecchio sito? 
Vorrei partire da quello per progettare il nuovo

**Comment by aborruso** [comment]
> Vorrei partire da quello per progettare il nuovo

Lo devi temporaneamente fare tornare come prima.

**Comment by gbvitrano** [comment]
ok grazie,  mi faccio gli screenshot del vecchio sito cosi non devo cambiare ogni 5 minuti :-)

**Reply by aborruso** [reply]
Secondo me, puoi anche non tenerne conto, e partire da zero ;)

**Comment by dennisangemi** [comment]
@gbvitrano già mi piace tantissimo! (molto più di quello "vecchio")

![image](https://user-images.githubusercontent.com/77018886/143774589-a282a44d-1391-4a78-8370-dcce04571bb4.png)

**Reply by gbvitrano** [reply]
> @gbvitrano già mi piace tantissimo! (molto più di quello "vecchio")
> 
> ![image](https://user-images.githubusercontent.com/77018886/143774589-a282a44d-1391-4a78-8370-dcce04571bb4.png)

mi passo il tempo e prendo confidenza con le varie pagine, prima non avevo accesso al lato admin e non conoscevo la struttura. :-)

**Reply by aborruso** [reply]
ma il vecchio non lo dovete guardare ;)

**Comment by gbvitrano** [comment]
Test con slider per articoli in evidenza

![Screenshot 2021-11-28 at 23-37-13 Open Data Sicilia – un'iniziativa civica che si propone di far conoscere e diffondere la ](https://user-images.githubusercontent.com/20488693/143789047-27dd71f4-d0e0-4f9f-9745-4bcde0dd8a93.png)


Test no slider

![Screenshot 2021-11-28 at 23-35-15 Open Data Sicilia – un'iniziativa civica che si propone di far conoscere e diffondere la ](https://user-images.githubusercontent.com/20488693/143789085-21fafc1e-79d3-44ab-b5fb-c84c62ba24a3.png)

**Comment by gbvitrano** [comment]
In molti articoli le immagini sono saltate, bisogna aprire gli articoli e ricaricarli, qualcuno l'ho fatto ma ne restano ancora tanti

**Reply by dennisangemi** [reply]
forse questa osservazione dovrebbe diventare una issue

**Reply by gbvitrano** [reply]
fatto  #26

**Comment by gbvitrano** [comment]
![Acquisisci schermata Web_29-11-2021_31945_opendatasicilia it](https://user-images.githubusercontent.com/20488693/143798967-a32b0015-19d1-4b8f-9402-69906d127cc7.jpeg)
